seedbuilder/areas.ori

```diff
@@ -1720,9 +1749,13 @@ home: UpperGinsoTree 528 468
 		standard GinsoKey Bash Health=3
 		standard GinsoKey DoubleJump Health=3
 		standard GinsoKey ChargeJump Health=3
-		standard GinsoKey ChargeJump AirDash
```

Comment:
Why did you remove this path? If it's too precise, it should go into expert. Cjump to the KS, then airdash to the right to safety.

Reply:
Yeah it did seem a bit precise when I first tried it. I think I did forget to put it to expert tho, so that's my bad. If you believe that standard is still fine for this then I can put it there instead.